docs/CONNECTOR_PROTOCOL.md

```diff
@@ -136,16 +268,48 @@ In addition to the connector index `.elastic-connectors`, we have an additional
 #### Elasticsearch mappings for `.elastic-connectors-sync-jobs`:
 ```
 "mappings" " {
+  "_meta" : {
+    "version" : 1
+  },
   "properties" : {
+    "completed_at" : { "type" : "date" },
```

Comment:
Is it the convention to have the properties sorted?

Reply:
It is in the Kibana codebase. And that's where we first defined these mappings, so this doc has been alphabetically sorted. I'm not tied to it here, if you'd rather that we re-order them, but keeping them ordered consistently across all our sources makes it easier to copy-paste from one codebase to the next and not have to manage weird diffs.

Reply:
I personally prefer those fields to be logically ordered. e.g. It's strange to see `completed_at` at the top and `started_at` at the bottom, `indexed_documents_count` and `deleted_documents_count` are supposed to be together.

Hi @sphilipse , any particular reason to sort them alphabetically?